webui/src/components/Animation/Spinner.vue

```diff
@@ -1,6 +1,7 @@
 <template>
     <div class="loading-spinner-default">
-        <div/><div/><div/><div/><div/><div/><div/><div/><div/><div/><div/><div/>
+        <div></div><div></div><div></div><div></div><div></div><div></div>
```

Comment:
Why do we need duplicate elements?

Reply:
Because we use it in index.html before loading vue. And this component we use as usual